readthedocs/settings/base.py

```diff
@@ -749,6 +749,15 @@ def DOCKER_LIMITS(self):
         'HEAD',
     ]
 
+    CORS_URLS_ALLOW_ALL_REGEX = [
```

Comment:
Could use a comment here.

```suggestion
    # URLs to allow CORS to read from unauthed.
    CORS_URLS_ALLOW_ALL_REGEX = [
```